spec.emu

```diff
@@ -74,7 +74,9 @@ contributors: Nicolò Ribaudo
           1. If _P_ is a Symbol, then
             1. Return ! OrdinaryGet(_O_, _P_, _Receiver_).
           1. <ins>Let _m_ be _O_.[[Module]].</ins>
-          1. <ins>If _m_ is not a Cyclic Module Record, or both _m_.[[Status]] is ~linked~ and AnyDependencyNeedsAsyncEvaluation(_m_) is *false*, then</ins>
+          1. <ins>If _m_ is not a Cyclic Module Record, then</ins>
+            1. <ins>Perform ? EvaluateSync(_m_).</ins>
+          1. <ins>Otherwise if _m_.[[Status]] is not ~evaluated~ and ! ReadyForSyncExecution(_m_) is *true*, then</ins>
```

Comment:
if ReadyForSyncExecution(m) is false, then what will this AO return? The (possibly) uninitialized binding and lead to TDZ error (for `export let`)?

Reply:
Yes, in the case of it not being ready, the namespace will expose TDZ or undefined for non-TDZ bindings, per the note.

Reply:
This is also what happens with cycles without `defer`:
```js
// entrypoint
import "dep";
export let a = 1;
```
```js
// dep
import * as ns from "entrypoint";
ns.a; // TDZ error
```